src/components/svg/Icons.tsx

```diff
@@ -661,6 +661,90 @@ export const ManyDocumentsIcon = ({ height = "24", width = "24", color = "curren
   );
 };
 
+export const FacebookIcon = ({ height = "20", width = "24", color = "#0171B8" }: IconProps) => {
+  return (
+    <svg
+      aria-label="Facebook Icon"
+      role="img"
+      xmlns="http://www.w3.org/2000/svg"
+      width={width}
+      height={height}
+      viewBox={`0 0 ${height} ${width}`}
```

Comment:
don't change the view box as this is the SVG's internal sizing - it isn't the size of the SVG itself, that is controlled by the width and height properties